tests/test_gen2convert.py

```diff
@@ -116,6 +118,41 @@ def testRawFilters(self):
         rawI = self.butler.get("raw", instrument="HSC", exposure=903986, detector=16)
         self.assertEqual(rawI.getFilter().getName(), "i")
 
+    def testCuratedCalibrations(self):
+        """Test that defects, the camera, and the brighter-fatter kernel were
+        added to the Gen3 registry.
+        """
+        originInfo = DatasetOriginInfoDef(["raw", "calib"], [])
+        rowsWithCalibs = list(
```

Comment:
Aren't these more like column names?

This is a nit pick, the term `WithCalibs` and `WithoutCalibs` is a bit confusing.  It seems more like `AllDatasets` and `RawOnly`.  Those may not be any better.  Take this with a grain of salt.

Reply:
They really are rows, but I agree it's not clear at all what they're rows of (the answer is a query that returns something like a big data ID that has keys for all of the given dataset types). I think it's best to leave the names and add some comments.

Reply:
OK